src/assets/usage-examples-code/graphics/clear_screen/clear_screen-background-color.cs

```diff
@@ -0,0 +1,10 @@
+using static SplashKitSDK.SplashKit;
+
+OpenWindow("House Drawing", 800, 600);
+
+ClearScreen(ColorBlue());
+RefreshScreen(60);
+Delay(5000);
+
+
```

Comment:
Remove the extra line space between the `Delay` function and the `CloseAllWindows` function to ensure consistent code formatting.